.appveyor.yml

```diff
@@ -62,6 +62,14 @@ install:
       If (!(Test-Path cpdksetup.exe )) {
           appveyor DownloadFile "https://download.microsoft.com/download/1/7/6/176909B0-50F2-4DF3-B29B-830A17EA7E38/CPDK_RELEASE_UPDATE/cpdksetup.exe"
       }
+  - wget https://ftp.gnu.org/gnu/gengetopt/gengetopt-2.23.tar.xz
+  - tar xJf gengetopt-2.23.tar.xz
+  - cd gengetopt-2.23
+  - bash -c "exec 0</dev/null && ./configure --prefix=/ || cat config.log"
+  - bash -c "exec 0</dev/null && make"
+  - cd ..
```

Comment:
Looking at the above block, I see installation of other dependencies (openpace, zlib) is done in `c:\` directly, which makes the paths slightly nicer. Whats the reason for installing the gengeopt in different path? Can the `appveyor DownloadFile` be used (I assume it can do some caching)?